src/components/Patient/Utils.ts

```diff
@@ -17,3 +17,16 @@ export function isPatientMandatoryDataFilled(patient: PatientModel) {
     patient.blood_group
   );
 }
+export const isPatientDischarged = (
+  patientData: PatientModel,
+  facilityId: string | undefined,
+) => {
+  if (!patientData.is_active) {
+    return true;
+  }
+  const patientFacilityId = patientData.facility_object?.id;
+  if (!patientFacilityId || !facilityId) {
+    return false;
+  }
```

Comment:
1. why do we have to check if facility ID matches?
2. Shouldn't we instead check if the `last_consultation.discharge_date` has a value or not?